Frontend service

lin-lin-enko opened the pull request.

lin-lin-enko committed:
Created basic structure and landing page

lin-lin-enko committed:
Moved frontend to next app

lin-lin-enko committed:
Designed filter component, sorting component, age selection, search bar; did minor design changes in header (animated svg and s.o.)

Yordaniss commented on .vscode/settings.json:
```diff
@@ -0,0 +1,3 @@
+{
+  "editor.acceptSuggestionOnEnter": "on"
```
not needed, because this is just for users, who are using vscode. to prevent commit this file, you could add 

`/.vscode` to `.gitignore` file

Yordaniss replied:
please remove this file from commit

Yordaniss commented on package.json:
```diff
@@ -7,7 +7,7 @@
   "dependencies": {
     "mongodb": "^4.8.1",
-    "next": "latest",
+    "next": "^13.0.3",
```
we should keep an eye, if these changes break our code

Yordaniss commented on pages/components/searchManagement/filterDropdown/FilterDropdown.js:
```diff
@@ -0,0 +1,20 @@
+      dropdownList={{
+        title: "Choose category:",
+        list: [
+          // "Indoor",
```
no needed?

Yordaniss commented on pages/fonts/comfortaa/misc/FONTLOG-3628.txt:
```diff
@@ -0,0 +1,110 @@
+	FONTLOG for Comfortaa:
```
is that necessary to keep this file?

Yordaniss commented on pages/fonts/comfortaa/misc/OFL-7f39.txt:
```diff
@@ -0,0 +1,94 @@
+Copyright (c) 2010, Johan Aakerlund (aajohan@gmail.com),
```
is that necessary to keep this file?

Yordaniss commented on pages/fonts/josefin sans/OFL.txt:
```diff
@@ -0,0 +1,93 @@
+Copyright 2010 The Josefin Sans Project Authors (https://github.com/ThomasJockin/JosefinSansFont-master), with Reserved Font Name "Josefin Sans".
```
is that necessary to keep this file?

Yordaniss commented on pages/fonts/josefin sans/README.txt:
```diff
@@ -0,0 +1,77 @@
+Josefin Sans Variable Font
```
is that necessary to keep this file?

Yordaniss commented on pages/fonts/motley forces/info.txt:
```diff
@@ -0,0 +1,2 @@
+﻿license: Freeware
```
is that necessary to keep this file?

Yordaniss commented on pages/fonts/varela-round-font/misc/OFL.txt:
```diff
@@ -0,0 +1,93 @@
+Copyright (c) 2011, Joe Prince, Admix Designs (http://www.admixdesigns.com/),
```
is that necessary to keep this file?

Yordaniss reviewed (changes requested):
@lin-a-chen  

- in documentation is described, that files in pages should be used as pages for routes. Logic for "components" we should move to root directory. Also new fonts is barely pages. They are rather peaces of library. I think we should also move this files to `lib` directory to keep structure clean
- please check if we also need empty files. For example `pages/media-queries/_Index.scss`
- do we need this file? `public/images/2109.i032.019.S.m005.c13.isometric girls room illustration [Converted].ai`

lin-lin-enko committed:
Changed project file structure, removed or/and added unnecessery files to gitignore

lin-lin-enko committed:
fixed dropdown code

Did changes in .js and .scss files to fix errors in dropdown design and selection logic

lin-lin-enko committed:
fixed svg scale issue

Path inside svg didn't take the whole svg elements' space

Yordaniss reviewed (approved):
We could try to merge

lin-lin-enko committed:
Fixed header layout

Fixed navbar display and text-shadow, fixed mobile layout and animation in header

lin-lin-enko committed:
Resolved the conflicts
